packages/react-devtools-shared/src/__tests__/inspectedElement-test.js

```diff
@@ -2099,7 +2099,7 @@ describe('InspectedElement', () => {
     await utils.actAsync(() => {
       const container = document.createElement('div');
       container.innerHTML = '<div></div>';
-      ReactDOMClient.hydrateRoot(container).render(<Example />);
+      ReactDOMClient.hydrateRoot(container, <Example />);
```

Comment:
My initial reaction was that I'm glad I'm not the only one who gets this wrong. [But it was me 😅 ](https://github.com/facebook/react/pull/22363/files#diff-11c09cf7ec0ed466fcabb7f3abc798d407062cacc789d56d89f2c56c1584d19fR1992)